frontend/components/mui-elements/mui-button.tsx

```diff
@@ -0,0 +1,46 @@
+// TODO: Use the style overwrite in the theme file
+// Once done, this file will be deprecated
+
+import React from 'react';
+import { alpha, Button, styled } from '@mui/material';
+
+interface Props {
+    muiButtonType: 'primary' | 'secondary', 
+    onClick: () => void,
+    type?: 'submit'
```

Comment:
instead of declaring everything here, you could reused original MaterialUi buttons' props type, and then pass the props directly to the button. It will allow us to customize it without having to re-add the props in the list (ex: if I want to use another variant)

Reply:
I can open another PR to update the MUI button style directly in the theme config file. You have good point here, I wasn't used to use library type in the past. It's a good practice indeed! Since this file is meant to be deprecated, I won't do the changes

Reply:
Both way are fine I think, but MUI enforce the theming indeed 😄 But what I actually meant is something like this:
```suggestion
    muiButtonProps: ButtonProps
```

and in the return: 
```js
<StyledButton 
            variant='contained'
            disableRipple
            disableElevation
            ...props.muiButtonProps
        >
```
It allows us to change the variant for example while keeping the same code, or use other MuiProps without having to add them everytime to the list